web/src/Pages/ProgrammeView/programmeView.tsx

```diff
@@ -1286,6 +1286,18 @@ const ProgrammeView = () => {
     calculations.constantVersion = data.constantVersion;
   }
 
+  const getFileName = (filepath: string) => {
+    const index = filepath.indexOf('?');
+    if (index > 0) {
+      filepath = filepath.substring(0, filepath.indexOf('?'));
```

Comment:
Use index here instead of filepath.indexOf('?')

Reply:
fixed